var/spack/repos/builtin/packages/hybridsim/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hybridsim(MakefilePackage):
+    """
+    HybridSim provides cycle-accurate simulation of a non-volatile
+    memory system augmented with a DRAM based cache. It uses DRAMSim2
+    for the DRAM model and NVDIMMSim for the non-volatile memory model
+    """
+
+    homepage = "https://github.com/jimstevens2001/HybridSim"
+    git = "https://github.com/jimstevens2001/HybridSim"
+
+    version('2.0.1', url="https://github.com/jimstevens2001/HybridSim/archive/v2.0.1.tar.gz",
+            sha256="57b82ac929acd36de84525e4d61358f1ab6532f5b635ca3f560e563479921937")
+
+    depends_on("dramsim2")
+    depends_on("nvdimmsim")
+    patch("makefile.patch", when="@2.0.1")
+
+    def install(self, spec, prefix):
+        install_tree(".", prefix)
+
+    def build(self, spec, prefix):
+        symlink(spec["dramsim2"].prefix, "DRAMSim2")
+        symlink(spec["nvdimmsim"].prefix, "NVDIMMSim")
```

Comment:
Why is this necessary?

Reply:
I'm not the author of the package. That's just how it's set up... I would happily change the package to work more sensibly if I owned it. It expects you to either symlink or copy those folders into its folder.